src/Analyzers/Resources.resx

```diff
@@ -153,4 +153,10 @@
   <data name="DurableClientBindingAnalyzerTitle" xml:space="preserve">
     <value>[DurableClient] must be used with DurableTaskClient</value>
   </data>
+  <data name="CancellationTokenOrchestrationAnalyzerMessageFormat" xml:space="preserve">
+    <value>Orchestration '{0}' is using a non-supported CancellationToken binding</value>
+  </data>
+  <data name="CancellationTokenOrchestrationAnalyzerTitle" xml:space="preserve">
+    <value>Function CancellationToken binding is not supported for OrchestrationTrigger</value>
```

Comment:
Should we use the description that's in the in proc analyzer (pasted below) to keep it consistent? This description might be simpler.

> CancellationToken should not be used as an orchestrator function parameter.

Reply:
Updated :)1